org.jacoco.report/src/org/jacoco/report/xml/report.dtd

```diff
@@ -44,6 +44,8 @@
 <!ELEMENT class (method*, counter*)>
   <!-- fully qualified VM name -->
   <!ATTLIST class name CDATA #REQUIRED>
+  <!-- name of the corresponding source file -->
+  <!ATTLIST class sourcefilename CDATA #IMPLIED>
```

Comment:
My eyes are bleeding a bit as a reaction on readability of this attribute name. But this is consistent with already existing other attributes that don't use CamelCase, what a pity that XML is case sensitive.

Wondering if better to keep it consistent with Java API or simplify into e.g. `source`, `sourcefile`? As a note - attribute of class denoting name of source file has name `SourceFile` in JVMS.

Reply:
I had the same thoughts but then decided for a consistent naming:

1) In our API the attribute is also called sourceFileName (ok in camel case)
2) Also it is conistent with the XML schema itself: It makes a reference to sourcefile/@name

Reply:
Ok